.github/workflows/unit_tests.yaml

```diff
@@ -288,7 +288,7 @@ jobs:
       shell: runuser -u spack-test -- bash {0}
```

Comment:
I think somewhere in this file you'll also want to `apt` or `dnf` install CVS, otherwise the tests will get skipped and there won't be any coverage.